restonomer-docs/source/restonomer_transformations.md

```diff
@@ -278,7 +278,46 @@ The transformed response will now have the columns with the desired data types:
 
 ```json
 {
-  "col_A": 5,
+  "col_A": 1,
   "col_B": "[{'ZipCodeType':'STANDARD','Zipcode':704}]"
 }
 ```
+
+## ChangeColumnCase
+
+It lets the user change the case of column names.
+
+This transformation expects user to provide below inputs:
+
+| Input Arguments | Mandatory | Default Value | Description                                      |
+|:----------------|:---------:|:-------------:|:-------------------------------------------------|
+| case-type       |    Yes    |       -       | Case to which the column needs to transformed    |
+
+For example, consider we have below restonomer response in json:
+
+```json
+{
+  "col_a": "1",
+  "COL_B": "2"
+}
+```
+
+Now, suppose the requirement is to transform case of COL_B to lowercase:
```

Comment:
This transformation will be applied to all columns, so do not mention any specific column name here as that can give the illusion that it is meant for a specific column only.

Reply:
resolved